server-mirror-git/src/main/java/com/linecorp/centraldogma/server/internal/mirror/AbstractGitMirror.java

```diff
@@ -723,6 +724,16 @@ static void updateRef(org.eclipse.jgit.lib.Repository jGitRepository, RevWalk re
         }
     }
 
+    private String getRemoteBranchName(Ref headBranchRef) {
+        if (remoteBranch() != null && !remoteBranch().isEmpty()) {
+            return remoteBranch();
+        }
+        if (headBranchRef.getName().startsWith(Constants.R_HEADS)) {
+            return headBranchRef.getName().substring(Constants.R_HEADS.length());
+        }
+        return "";
```

Comment:
Let's just return `headBranchRef.getName()` because `Miiror abbrId, remote-uri# to the repository` is a bit awkward.

Reply:
Hi @minwoox, you mean line 734 🤔

Reply:
Yes. 😊

Reply:
Pushed a small commit which:
- Address this comment: https://github.com/line/centraldogma/pull/934#discussion_r1594973369
- Add single quotes around the `remoteUri#branchName`